src/request/actions/ActionCreator.ts

```diff
@@ -30,3 +34,17 @@ export function createActionCreator<TPayload>(type: string): IActionCreator<TPay
     
     return typedActionCreator;
 }
+
+export function createSimpleActionCreator(type: string): ISimpleActionCreator {
+    const actionCreator = () => {
+        return {
+            type: type        
+        }
+    };
+    
+    const typedActionCreator = <ISimpleActionCreator>actionCreator;
+    
+    typedActionCreator.type = type;
+    
+    return typedActionCreator;
+}
```

Comment:
Yes.  We could have used a singleton action instance, but for the sake of consistency with payload-bearing actions it made sense to keep the same action-factory pattern.